extension-jaxrs/src/test/resources/io/smallrye/openapi/runtime/scanner/responses.exception-mapper-multiple-response-generation.json

```diff
@@ -0,0 +1,44 @@
+{
+  "openapi" : "3.0.3",
```

Comment:
Should this not change to 3.1.0 now everywhere ?

Reply:
Not yet, Microprofile 3.1 does not yet support OAS 3.1 :-)

Reply:
A, ok, I thought that is the case. No worries then